graph_snyk_cve_sync.py

```diff
@@ -360,19 +458,13 @@ def disable_snyk_run(self, date):
     def run_snyk_sync(self):
         """Entrypoint for the snyk cve sync process."""
         logger.info("Running Snyk Sync".center(50, '-'))
-        utc_now = datetime.utcnow()
-        # TBD. comparison with prev day's feed for ingestion
-        date_yest = (utc_now - timedelta(days=1)).strftime('%d%m%y')
-        logger.info(utc_now)
-        if self.disable_snyk_run(utc_now):
+        logger.info(self.utc_now)
+        if self.disable_snyk_run(self.utc_now):
             logger.info("Snyk sync to run only once a day. Won't run at this hour.")
             return
-        data = self.helper.read_data_from_s3(utc_now.strftime('%d-%m-%y'), "snyk-feed/")
-        data_yest = self.helper.read_data_from_s3(date_yest, "snyk-feed/")
-        # TBD add logic for comparison and use delta info for cve ingestion
-        logger.debug(data_yest)
-        if data:
-            self.extract_data_from_feed(data)
+
+        if self.snyk_data:
+            self.extract_data_from_feed()
             self.insert_cves()
         else:
             logger.info("No data found. Snyk sync aborted.")
```

Comment:
This will continue and return success even if there is no data

Reply:
noted

Reply:
done

Reply:
Dont see this change yet

Reply:
store data is an independent function. It doesnt have any relation with the success or failure of ingestion